org.aposin.licensescout.core/src/test/java/org/aposin/licensescout/license/LicenseUtilStatisticsTest.java

```diff
@@ -0,0 +1,161 @@
+/**
+ * Copyright 2019 Association for the promotion of open-source insurance software and for the establishment of open interface standards in the insurance industry (Verein zur Förderung quelloffener Versicherungssoftware und Etablierung offener Schnittstellenstandards in der Versicherungsbranche)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.aposin.licensescout.license;
+
+import java.util.ArrayList;
+import java.util.Collection;
+
+import org.aposin.licensescout.archive.Archive;
+import org.aposin.licensescout.archive.ArchiveType;
+import org.aposin.licensescout.exporter.GeneralStatistics;
+import org.aposin.licensescout.exporter.IDetectionStatusStatistics;
+import org.aposin.licensescout.exporter.ILegalStatusStatistics;
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * Unit tests for {@link LicenseUtil}.
+ * 
+ * 
+ * @see LicenseUtil
+ * @see LicenseUtilGetMatchedVersionFromLineTest
+ * @see LicenseUtilDetectLicensesTest
+ *
+ */
+public class LicenseUtilStatisticsTest {
+
+    /**
+     * 
+     */
+    @Test
+    public void testCalculateGeneralStatisticsEmptyList() {
+        final Collection<Archive> archives = new ArrayList<Archive>();
+        final GeneralStatistics generalStatistics = LicenseUtil.calculateGeneralStatistics(archives);
+        Assert.assertEquals("candidateLicenseFileCount", 0, generalStatistics.getCandidateLicenseFileCount());
+        Assert.assertEquals("totalArchiveCount", 0, generalStatistics.getTotalArchiveCount());
+    }
+
+    /**
+     * Test for {@link LicenseUtil#calculateGeneralStatistics(Collection)}.
+     */
+    @Test
+    public void testCalculateGeneralStatisticsRealList() {
+        final Collection<Archive> archives = new ArrayList<Archive>();
+        for (int i = 1; i <= 3; i++) {
```

Comment:
Extract to a variable the number of archives that you are generating an use in the last assertion to be consistent.

```suggestion
        final int numberOfArchives = 3;
        for (int i = 1; i <= numberOfArchives ; i++) {
```

Reply:
Resolved as suggested.

Reply:
Sorry, the change seems to have been lost somehow. Should now be fixed.